core/src/main/scala/kafka/admin/AdminUtils.scala

```diff
@@ -608,7 +608,7 @@ object AdminUtils extends Logging with AdminUtilities {
   def fetchAllChildEntityConfigs(zkUtils: ZkUtils, rootEntityType: String, childEntityType: String): Map[String, Properties] = {
     def entityPaths(zkUtils: ZkUtils, rootPath: Option[String]): Seq[String] = {
       val root = rootPath match {
-        case Some(path) => rootEntityType + '/' + rootPath
+        case Some(path) => rootEntityType + '/' + path
```

Comment:
@ijuma Actually, the code above doesn't have any impact unless someone goes and explicitly deletes config change notifications from Zookeeper. I can't find any code that deletes change notification znodes from Zookeeper - do you know if that is intentional?